content/html/concepts/attributes/terms/src.md

```diff
@@ -0,0 +1,48 @@
+﻿---
+Title: 'src'
+Description: 'An HTML attribute used to specify the location of a digital resource.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Attributes'
+  - 'Elements'
+  - 'Style'
+CatalogContent: 
+   - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The  **`src`** attribute is  used to specify the location of a digital source, which is typically a URL. It is required for HTML elements like [`<img>`](https://www.codecademy.com/resources/docs/html/images) so that they can appear on a website or application. 
```

Comment:
```suggestion
The  **`src`** attribute is  used to specify the location of a digital source, which is typically a URL. It is required for HTML elements like [`<img>`](https://www.codecademy.com/resources/docs/html/elements/img) so that they can appear on a website or application. 
```